pkg/logicalbackup/logicalbackup.go

```diff
@@ -689,21 +753,48 @@ func (b *LogicalBackup) QueueBasebackupTables() {
 
 func (b *LogicalBackup) Run() {
 	b.waitGr.Add(1)
-	go b.startReplication()
+	go b.LogicalDecoding()
 
 	log.Printf("Starting %d background backupers", b.cfg.ConcurrentBasebackups)
 	for i := 0; i < b.cfg.ConcurrentBasebackups; i++ {
 		b.waitGr.Add(1)
-		go b.BackgroundBasebackuper()
+		go b.BackgroundBasebackuper(i)
 	}
 
+	listenAndServeAttempt := func() (bool, error) {
```

Comment:
do we really need to retry starting `ListenAndServe` ? the usual case when it fails is when the port is in use by the other process